app/code/Magento/Customer/Model/ForgotPasswordToken/ConfirmByToken.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types=1);
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Customer\Model\ForgotPasswordToken;
+
+use Magento\Customer\Api\CustomerRepositoryInterface;
+
+/**
+ * Class ConfirmByToken
```

Comment:
Repeating class name in the comment is redundant. Clear responsibility description is sufficient.

Reply:
Cannot agree, but fixed.